src/OrchardCore/OrchardCore.FileStorage.AmazonS3/AwsStorageOptionsBase.cs

```diff
@@ -0,0 +1,25 @@
+using Amazon.Extensions.NETCore.Setup;
+
+namespace OrchardCore.FileStorage.AmazonS3;
+
+public abstract class AwsStorageOptionsBase : IAwsStorageOptions
+{
+    /// <inheritdoc/>
```

Comment:
these useless comments can be removed

Reply:
These make the comments of the interface appear for the properties if you inspect the class in IntelliSense; otherwise, they wouldn't have any documentation. But I removed the interface since then.